elasticdl/proto/elasticdl.proto

```diff
@@ -41,6 +47,9 @@ message Task {
 
     // Whether this is training or evaluation task.
     TaskType type = 7;
+
+    // Extended task configuration as a key value list
```

Comment:
"a key value list" -> "a list of key-value pairs"

Reply:
Good suggestion, updated.